app/assets/stylesheets/ama_layout/layout_components/sections.scss

```diff
@@ -166,11 +169,29 @@
     }
   }
 
+  &--cerulean-bg{
+    padding: $base-padding;
+    background: $cerulean;
+    color: $white;
+    @include clearfix;
+
+    label{
+      color: $white;
+    }
+  }
+
   &--grey-border{
     border: 2px solid $stone;
   }
 }
 
+.section-content-half{
```

Comment:
I think this is more of a modifier of section-content, not the section-half block. I kept the full class name outside of the section-content class to follow the same convention we have above for the sections (like where section-half, section-full are on their own lines ). Should I move it to live inside of section-content?